src/js/components/Calendar/StyledCalendar.js

```diff
@@ -100,17 +131,79 @@ const daySizeStyle = (props) => {
   `;
 };
 
+const dayStyle = (props) => {
+  let backgroundObj;
+  let colorObj;
+  if (props.isSelected) {
+    backgroundObj = props.theme.calendar.day?.selected?.background || 'control';
+    colorObj = props.theme.calendar.day?.selected?.color;
+  } else if (props.inRange) {
+    // for backwards compatability, only apply this if caller hasn't specified
+    // range specific rounding
+    // if they have, background will be applied to StyledDayContainer
```

Comment:
so if the caller does something like 
```
      // range: {
      //   round: 'none',
      //   start: {
      //     round: {
      //       corner: 'left',
      //       size: 'full',
      //     },

```

then this will apply?

Reply:
This will apply when caller doesn't have anything applied under `range.round` (which allows for styling to be backwards compatible w existing behavior). Once they apply something in that object, it'll apply the styling to StyledDayContainer.

Reply:
okay thank you for clarifying